doc/rbd/rados-rbd-cmds.rst

```diff
@@ -182,20 +182,20 @@ For example::
 
 .. note::
 
-  * You can move an image to the trash even it has shapshot(s) or actively 
+  * You can move an image to the trash even it has snapshot(s) or actively 
     in-use by clones, but can not be removed from trash.
 
-  * You can use *--delay* to set the defer time (default is 0), and if its 
-    deferment time has not expired, it can not be removed unless you use 
-    force.
+  * You can use *--expires-at* to set the defer time (default is now), and 
```

Comment:
i think the `delay = 0` is more accurate.

Reply:
@zhongyimao i mean, a number "0" instead of "now" is better in this context. so i'd suggest keep as it is.

Reply:
@zhongyimao sorry, i was wrong. see https://github.com/ceph/ceph/commit/b9e4aa94b77ab9fca803df330a8e5f31e8de0f00#diff-b6fb76bfc588ba6ebd1506f4cd6403efR1584 . could you quote "now" like ``` ``now`` ``` ?

Reply:
Sorry, I'm wondering if the "now"  you said is pointed https://github.com/ceph/ceph/commit/b9e4aa94b77ab9fca803df330a8e5f31e8de0f00#diff-b6fb76bfc588ba6ebd1506f4cd6403efR1584 or `+  * You can use *--expires-at* to set the defer time (default is now)`?   thank you.

Reply:
ok, I got it, thanks.